### ISSUE TITLE
Book & Go - Web and mobile issue

### ISSUE THREAD

**Issue by GrosR** [clowdfish/ttg-frontend#180]
If a user clicks on a booking link - do we have any means to redirect the user afterwards to our site again?

**Comment by davidtran** [comment]
It opens a new booking site in new tab.

**Comment by GrosR** [comment]
yes, that´s what i mean... the user finds himself on a new tab and I don´t know if they´ll find their way back to our site .... that´s why we need integrated booking as soon as possible :-) 
Opening new tabs is an option - but we should check if there is some "redirect option" with the integrations - like we find on Paypal

**Comment by davidtran** [comment]
We can cache data in the browser and when user go back we load data from cache. However if user use our website in private mode we can't use the cache.

**Comment by GrosR** [comment]
I think we must check if the affiliate sites offer a redirect after booking - they should give the token back after they confirmed transaction
